cartography/intel/aws/ec2.py

```diff
@@ -607,6 +786,134 @@ def load_load_balancer_subnets(neo4j_session, load_balancer_id, subnets_data, aw
+            'LoadBalancerArn': 'string',
+            'Port': 123,
+            'Protocol': 'HTTP''HTTPS''TCP''TLS''UDP''TCP_UDP',
+            'Certificates': [
+                {
+                    'CertificateArn': 'string',
+                    'IsDefault': TrueFalse
+                },
+            ],
+            'SslPolicy': 'string',
+            'DefaultActions': [
+                {
+                    'Type': 'forward''authenticate-oidc''authenticate-cognito''redirect''fixed-response',
+                    'TargetGroupArn': 'string', # Used with forward
+                    'AuthenticateOidcConfig': {
+                        'Issuer': 'string',
+                        'AuthorizationEndpoint': 'string',
+                        'TokenEndpoint': 'string',
+                        'UserInfoEndpoint': 'string',
+                        'ClientId': 'string',
+                        'ClientSecret': 'string',
+                        'SessionCookieName': 'string',
+                        'Scope': 'string',
+                        'SessionTimeout': 123,
+                        'AuthenticationRequestExtraParams': {
+                            'string': 'string'
+                        },
+                        'OnUnauthenticatedRequest': 'deny''allow''authenticate',
+                        'UseExistingClientSecret': TrueFalse
+                    },
+                    'AuthenticateCognitoConfig': {
+                        'UserPoolArn': 'string',
+                        'UserPoolClientId': 'string',
+                        'UserPoolDomain': 'string',
+                        'SessionCookieName': 'string',
+                        'Scope': 'string',
+                        'SessionTimeout': 123,
+                        'AuthenticationRequestExtraParams': {
+                            'string': 'string'
+                        },
+                        'OnUnauthenticatedRequest': 'deny''allow''authenticate'
+                    },
+                    'Order': 123,
+                    'RedirectConfig': {
+                        'Protocol': 'string',
+                        'Port': 'string',
+                        'Host': 'string',
+                        'Path': 'string',
+                        'Query': 'string',
+                        'StatusCode': 'HTTP_301''HTTP_302'
+                    },
+                    'FixedResponseConfig': {
+                        'MessageBody': 'string',
+                        'StatusCode': 'string',
+                        'ContentType': 'string'
+                    }
+                },
+            ]
+        },
+    ],
+    """
+    ingest_listener = """
+    MATCH (elbv2:LoadBalancerV2{id: {LoadBalancerId}})
+    WITH elbv2
+    UNWIND {Listeners} as data
+        MERGE (l:Endpoint:ELBV2Listener{id: elbv2.id + toString(data.Port) +
+                toString(data.Protocol)})
+        ON CREATE SET l.port = data.Port, l.protocol = data.Protocol,
```

Comment:
Is `data.protocol` a list or string separated by a character?  I think neo4j supports lists on nodes with https://neo4j.com/docs/cypher-manual/current/functions/list/.

In general, to represent lists I think we should create new nodes and attach to them, but since this is a list of protocols it doesn't make much sense to create new nodes,   so it'd be most intuitive to represent these protocols with neo4j lists.

Reply:
`data.protocol` is a string. It must be **one of** `HTTP`, `HTTPS`, `TCP`, `TLS`, `UDP`, `TCP_UDP`